hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -289,6 +289,11 @@ public static class Config implements Serializable {
     )
     public List<String> transformerClassNames = null;
 
+    @Parameter(names = {"--enable-transformer-schema-validation"},
+        description = "If enabled, schema is validated for the transformed data against expected schema. "
+            + "Expected schema is provided by the transformer.")
+    public Boolean enableTransformerSchemaValidation = false;
```

Comment:
`boolean` since we don't expect this to be null

Reply:
Addressed